src/index.js

```diff
@@ -1,9 +1,13 @@
-const title = document.getElementById('title');
-const author = document.getElementById('author');
-const addBtn = document.getElementById('add');
 const list = document.getElementById('list');
+const main = document.getElementById('main');
 
-const listbooks = JSON.parse(localStorage.getItem('books'));
+const listMenu = document.getElementById('listMenu');
+const addNewMenu = document.getElementById('addNewMenu');
+const contactMenu = document.getElementById('contactMenu');
+
+let menu = '';
+
+const listbooks = localStorage.getItem('books') ? JSON.parse(localStorage.getItem('books')) : [];
 
 class Book {
   constructor(title, author) {
```

Comment:
- [ OPTIONAL ] Right now when I add books with the same `title` and delete only one of them, the rest of the same-title books will also be deleted, this is evident on page refresh and also in the localStorage. 

https://user-images.githubusercontent.com/45231276/182752256-cd75c1b1-7a99-4c24-8036-45b2c7c88a02.mp4


> There are two solutions to fix this issue 💡 : 

1) you can add an `id` property to a book object which should be unique. The `id` property could then be used to filter books. Since the `id` property is unique, only one book(**_`the correct/intended book`_**) will be deleted at a time. 👌

2) Another solution would be to prevent the addition of duplicate books in the `add()` method. 👌

⭕ I marked this as OPTIONAL (Since this is not a requirement for this exercise, it was supposed to be implemented beforehand), but please 🙏, It's **important** to work and solve this issue Because it was a main requirement in the first exercise [plain JavaScript with objects](https://github.com/microverseinc/curriculum-javascript/blob/main/books/m1_plain_js_objects_v1_1.md)

![image](https://user-images.githubusercontent.com/45231276/182751627-92bc30ee-d647-437d-80e3-75ede68de0b9.png)

Reply:
Thank you @Bondok6 , The id was not mentioned there and in real projects we will be getting it from database, I will make sure the next time I don't have that additional file from vs code.

**Thanks!**